pkg/utils/netlink.go

```diff
@@ -108,3 +119,40 @@ func bridgeTrunkVlanAdd(nlink Netlink, link netlink.Link, vlans []int) error {
 	}
 	return nil
 }
+
+// GetBridgeByLink returns linux bridge link if provided link belongs to any
+// if provided link is a part of a bond will return a bridge to which this bond belongs to
+func GetBridgeByLink(nLink Netlink, link netlink.Link) (netlink.Link, error) {
+	master, err := getMasterInterface(nLink, link)
+	if err != nil {
+		return nil, err
+	}
+	if master == nil {
+		return nil, nil
```

Comment:
why you don't return error if link don't have master? I see you handle it in the upper layer, but why not handle it here? same comment for line 145 and 149.

Reply:
I changed the function to return errors